llm/llama.cpp/generate.go

```diff
@@ -1,8 +1,10 @@
 package llm
 
 //go:generate git submodule init
-//go:generate git submodule update --force ggml
+//go:generate git submodule update --force ggml gguf
 //go:generate git -C ggml apply ../ggml_patch/0001-add-detokenize-endpoint.patch
 //go:generate git -C ggml apply ../ggml_patch/0002-34B-model-support.patch
 //go:generate cmake -S ggml -B ggml/build/cpu -DLLAMA_K_QUANTS=on
 //go:generate cmake --build ggml/build/cpu --target server --config Release
+//go:generate cmake -S gguf -B gguf/build/cpu -DLLAMA_K_QUANTS=on
+//go:generate cmake --build gguf/build/cpu --target server --config Release
```

Comment:
It might be better to split this into `generate_ggml` and `generate_gguf`. These `go:generate` instructions are piling up

Reply:
I'm gonna leave these for now just because the amount of files is growing a lot as of #459